src/main/java/ua/in/kp/controller/ProjectController.java

```diff
@@ -31,14 +33,25 @@ public ResponseEntity<ProjectResponseDto> createProject(
                 .createProject(createdProject), HttpStatus.CREATED);
     }
 
+    @GetMapping("/{id}")
+    public ResponseEntity<ProjectResponseDto> getProjectById(@PathVariable String id) {
+        return new ResponseEntity<>(projectService.getProjectById(id), HttpStatus.OK);
+    }
+
     @GetMapping()
     public ResponseEntity<Page<GetAllProjectsDto>> getAllProjects(Pageable pageable) {
         return new ResponseEntity<>(projectService.getAllProjects(pageable), HttpStatus.OK);
     }
+    @PutMapping("/{id}")
+    public ResponseEntity<ProjectResponseDto> updateProject(@PathVariable String id,
+            @Valid @RequestBody ProjectCreateRequestDto createdProject) {
```

Comment:
we can not update all fields in project, There should be created special DTO for update.